src/app/statistics/aggregating-collector.js

```
@@ -49,7 +54,12 @@ class AggregatingCollector implements EventCollector {
 
   async _sendEventsToDelegate (): Promise<any> {
     let eventsToSend = this._events.splice(0, this._accumulatorSize)
-    return this._delegate.collectEvents(...eventsToSend)
+
+    try {
+      await this._collector.collectEvents(...eventsToSend)
+    } catch (e) {
+      logger.error('Failed to send app metrics', e.stack)
```

This can catch all sorts of problems, including programming problems - undefined method. If we're not capturing this error in Sentry, we will not know about this!

We should check if it's known error from tequilapi - than it's ok to ignore it, but in other cases, we can't ignore errors.